std/php7/NativeArray.hx

```diff
@@ -21,6 +21,67 @@
  */
 package php7;
 
-extern class NativeArray implements ArrayAccess<Dynamic> {
+import haxe.extern.EitherType;
 
-}
\ No newline at end of file
+using php7.Global;
+
+
+/**
+	Native PHP array.
+**/
+@:coreType @:runtimeValue abstract NativeArray {
+	public inline function new()
+		this = untyped __php__("[]");
+
+	@:arrayAccess function get(key:Scalar):Dynamic;
+	@:arrayAccess function set(key:Scalar, val:Dynamic):Dynamic;
```

Comment:
Ah, this is the trick! I've been trying this for hours, but didn't come up with this one.